doc/crypto/api/ops/hashes.rst

```diff
@@ -73,6 +73,20 @@ Hash algorithms
 
     RIPEMD-160 is defined in :cite-title:`RIPEMD`, and also in :cite-title:`ISO10118`.
 
+.. macro:: PSA_ALG_AES_MMO_ZIGBEE
+    :definition: ((psa_algorithm_t)0x02000006)
+
+    .. summary::
+        The *zigbee* 1.0 hash function based on a Matyas-Meyer-Oseas (MMO) construction of AES-128.
```

Comment:
Editorial: “construction _on_ AES-128”? _over_ AES-128?

Reply:
Decided on 'using' - same as the full description that follows.